src/test/java/teammates/test/cases/webapi/GetAuthInfoActionTest.java

```diff
@@ -126,8 +134,38 @@ protected void testExecute() {
                 Const.ParamsNames.USER_ID, "idOfAnotherInstructor",
         }));
 
-        // TODO test CSRF token cookies
+        ______TS("Test adding CSRF token cookies");
+
+        loginAsInstructor("idOfInstructor1OfCourse1");
```

Comment:
CSRF token will be sent regardless of login status. So you should have test case when users is not logged-in?